app/src/main/kotlin/com/wire/android/ui/home/conversationslist/MentionScreen.kt

```diff
@@ -49,22 +60,26 @@ private fun MentionContent(
             header = { stringResource(id = R.string.mention_label_unread_mentions) },
             items = unreadMentions
         ) { unreadMention ->
-            MentionConversationItem(
-                mention = unreadMention,
+            ConversationItemFactory(
+                conversation = unreadMention,
                 eventType = EventType.UnreadMention,
-                onMentionItemClick = onMentionItemClick,
-                onConversationItemLongClick = onEditConversationItem
+                openConversation = { onMentionItemClick(unreadMention.id) },
+                onConversationItemLongClick = { onEditConversationItem(unreadMention.conversationType) },
+                openUserProfile = onOpenUserProfile,
+                onMutedIconClick = { openConversationNotificationsSettings(unreadMention.conversationType) },
```

Comment:
if we let ConversationItemFactory  openConversation, onConversationItemLongClick,onMutedIconClick emit the needed models we also get rid of the recomposition problem because we do not need to create those lambdas to extract the models we need from the model emitted by the ConversationItemFactory

Reply:
Fixed